packages/jupyter-chat/src/widgets/chat-widget.tsx

```diff
@@ -14,11 +14,12 @@ export class ChatWidget extends ReactWidget {
   constructor(options: Chat.IOptions) {
     super();
 
-    this.id = 'jupyter-chat::widget';
     this.title.icon = chatIcon;
     this.title.caption = 'Jupyter Chat'; // TODO: i18n
 
     this._chatOptions = options;
+    this.id = `jupyter-chat::widget::${options.model.name}`;
+    this.node.onclick = () => this.model.focusInput();
```

Comment:
Note that this change causes the input to essentially always be focused unless the user clicks out of the chat widget. I think this is fine, but want us to be aware of this change.

Reply:
Yes, actually the `WidgetTracker` relies on the [`FocusTracker`](https://github.com/jupyterlab/lumino/blob/main/packages/widgets/src/focustracker.ts), which needs to have an element focused to trigger the change of current widget.

This is mentioned in the PR description:
> EDIT: the chat input is focussed when clicking anywhere on the chat except on a focusable element.